docs/statistics.md

```diff
@@ -111,11 +119,31 @@ The time elapsed, in milliseconds, since the SRT socket has been created (after
 
 ### pktSentTotal
 
-The total number of sent data packets, including retransmitted packets. Available for sender.
+The total number of sent DATA packets, including retransmitted packets ([pktRetransTotal](#pktRetransTotal)). Available for sender.
+
+In case the `SRTO_PACKETFILTER` socket option is enabled (refer to [API.md](API.md)), this statistic counts sent packet filter control packets ([pktSndFilterExtraTotal](#pktSndFilterExtraTotal)) as well. Introduced in SRT v1.4.0.
 
 ### pktRecvTotal
 
-The total number of received packets, including retransmitted packets. Available for receiver.
+The total number of received DATA packets, including retransmitted packets ([pktRcvRetransTotal](#pktRcvRetransTotal)). Available for receiver.
```

Comment:
Per mechanism explanation: "This metric is counted for every DATA packet that has been received over the network".

Reply:
The same as with pktSent.